### ISSUE TITLE
Does instance sampler in create_transformation method automatically adds validation set.

### ISSUE THREAD

**Issue by abubakar12** [awslabs/gluon-ts#733]
Extended forecast tutorial mentions:
_The Transformation may not define an additional field in the dataset. However, it always needs to define how the datasets are going to be split in example windows during training and testing. This is done with the InstanceSplitter_ . 

1. Does instance splitter randomly cuts timeseries in training and validation parts. So the loss that is being calculated during the training is being optimized with the help of both training and validation sets. 
2. Do i need to manually give validation data during training. 

3.How much should be the size of timeseries in validation set. I had a pandas data frame with 100 rows and 34 columns, so this means i have 100 different time series with 34 months. When i manually splitted the timeseries in to 32 months for training , 1 month for validation and one month for testing i got the error during training.
**Exception: Reached maximum number of idle transformation calls**

4: Then i manually splitted the dataset in to 31 months for training, 2 months for validation and one month for testing, my error was gone. Is it necessary to have some fixed length of target variable in validation set.
